tastyworks/models/trading_account.py

```diff
@@ -89,7 +89,26 @@ def from_dict(cls, data: dict) -> List:
             res.append(acct)
 
         return res
-
+        
+    @classmethod
+    async def get_account_balance(cls, session, account):
```

Comment:
Do you think this would be better as a more general, `get_latest_balance()` instance method?

I don't think it needs to be a class method since it's not returning a new instance of the `TradingAccount` class.

Reply:
I'm not sure about bumping the version and changelog file.  If you don't mind, here is a link to the dates with a few additional endpoints.  https://paste.debian.net/1071812/

    async def get_account_balance(cls, session, account):
        url = f'{session.API_url}/accounts/{account.account_number}/balances'
    async def get_account_positions(cls, session, account):
        url = f'{session.API_url}/accounts/{account.account_number}/positions'
    async def get_account_live_orders(cls, session, account):
        url = f'{session.API_url}/accounts/{account.account_number}/orders/live'
    async def get_account_history(cls, session, account):
        url = f'{session.API_url}/accounts/{account.account_number}/transactions'

Reply:
👍 . They look good to me. If you could just rename them to take the 'account' part out, e.g. `get_account_history()` -> `get_history()` as they'll be methods on a `TradingAccount` class. E.g. `main_trading_account.get_history()` seems better than `main_trading_account.get_account_history()`

I don't mind doing the changelog and version bump if you do not wish to.

Could you also run `flake8` on the file (trading_account.py) as that's what is failing the Travis build.

Reply:
Here is another attempt... it should contain the updates you mentioned above. http://paste.debian.net/1071879/